scrapyd_k8s/launcher/k8s.py

```diff
@@ -26,6 +26,10 @@ def __init__(self, config):
         self._k8s_batch = kubernetes.client.BatchV1Api()
 
     def get_node_name(self):
+        deployment_name = os.getenv('MY_DEPLOYMENT_NAME')
+        pod_name = os.getenv('MY_POD_NAMESPACE')
+        if not deployment_name and not pod_name:
+            return "default"
         return "%s.%s" % (os.getenv('MY_DEPLOYMENT_NAME'), os.getenv('MY_POD_NAMESPACE'))
```

Comment:
What about?

```python
deployment = os.getenv('MY_DEPLOYMENT_NAME', 'default')
namespace = os.getenv('MY_NAMESPACE')
return ".".join([n for n in [pod_name, deployment_name] if n])
```

Also, looking at this, I wonder if it makes sense to put this in the Kubernetes manifest, and just have `SCRAPYD_K8S_NODE_NAME`, and build this a variable in the Kubernetes manifest.